Backend/lib/yudhishthira/src/Lib/Yudhishthira/Event/KaalChakra.hs

```diff
@@ -36,35 +40,37 @@ import qualified Lib.Yudhishthira.Types.UserData as DUserData
 
 data Handle m = Handle
   { getUserTags :: Id Yudhishthira.User -> m (Maybe [Text]), -- Nothing if user not found
-    updateUserTags :: Id Yudhishthira.User -> [Text] -> m ()
+    updateUserTags :: Id Yudhishthira.User -> [Text] -> m (),
+    createFetchUserDataJob :: Yudhishthira.UpdateKaalBasedTagsJobReq -> UTCTime -> m (),
+    createUpdateUserData :: Yudhishthira.RunKaalChakraJobReq -> Id Yudhishthira.Event -> UTCTime -> m ()
   }
 
 --  which is log level in PROD?
 skipUpdateUserTagsHandler :: (Monad m, Log m) => Handle m
 skipUpdateUserTagsHandler =
   Handle
     { getUserTags = \userId -> logInfo ("Skip update user tags in DB selected: userId: " <> show userId) >> pure (Just []),
-      updateUserTags = \userId updatedTags -> logInfo $ "Skip update user tags in DB selected: userId: " <> show userId <> "; updated tags: " <> show updatedTags
+      updateUserTags = \userId updatedTags -> logInfo $ "Skip update user tags in DB selected: userId: " <> show userId <> "; updated tags: " <> show updatedTags,
+      createFetchUserDataJob = \updateTagData _scheduledTime -> logInfo $ "Skip generateUserData job for: " <> show updateTagData,
+      createUpdateUserData = \kaalChakraData eventId _scheduledTime -> logInfo $ "Skip updateTag job for: " <> show kaalChakraData <> "; for event: " <> show eventId
     }
 
 kaalChakraEvent ::
   (BeamFlow m r, CH.HasClickhouseEnv CH.APP_SERVICE_CLICKHOUSE m) =>
-  Handle m ->
```

Comment:
kaal-chakra app did not compiled. I know, that we don't use it now, but anyway all backend code should be compiled